oot.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/bash
+
+while getopts ':c:' OPT; do
+        case ${OPT} in
+        c)
+		case ${OPTARG} in
+		clang|gcc|linaro_gcc)
+			export _compiler=${OPTARG}
+			;;
+		*)
+			echo "Compiler '${OPTARG}' unknown or not implemented"
+			exit
+			;;
+		esac
+		;;
+        ?)
+                echo && echo "Incorrect option: ${OPT}"
```

Comment:
This always prints `Incorrect option: ?`. I don't really like the extra empty line before through `echo &&`, what's your usecase for it?

Reply:
Fixed.

Reply:
Ah when the argument is not recognized they pass it through `OPTARG` :slightly_smiling_face: